src/middleware/authentication/middleware.ts

```diff
@@ -1,29 +1,31 @@
 import type { NextFunction, Request, Response } from "express";
-import { introspectToken } from "@lib/idp/introspectToken.js";
 import {
   getIntrospectionCache,
   setIntrospectionCache,
 } from "@lib/idp/introspectionCache.js";
-import { logEvent } from "@src/lib/auditLogs.js";
+import { introspectToken } from "@lib/idp/introspectToken.js";
+import { logEvent } from "@lib/logging/auditLogs.js";
 
 export async function authenticationMiddleware(
   request: Request,
   response: Response,
   next: NextFunction,
-) {
+): Promise<void> {
   const accessToken = request.headers.authorization?.split(" ")[1];
   const formId = request.params.formId;
 
   if (!accessToken) {
-    return response.sendStatus(401);
+    response.sendStatus(401);
+    return;
```

Comment:
Just curious why the change.  Not that I'm opposed, but was wondering.

Reply:
Sure thing! With the introduction of the new `ApiOperation` pattern for implementing new API paths I had to create types that would conform to what Express expects. They need operation handlers to return either `void` or `Promise<void>` so the old `return response.something` was not working anymore.
Also when discussing this with Bryan, he mentioned that having the `return` on a different line also makes it clear that we are stopping the execution on purpose.

Reply:
Gotcha, that makes sense - thanks!